.pre-commit-config.yaml

```diff
@@ -5,3 +5,16 @@
     - id: git-check  # Configure in .gitattributes
     - id: git-dirty  # Configure in .gitignore
     - id: shellcheck
+- repo: https://github.com/pre-commit/mirrors-mypy
+  sha: v0.701
+  hooks:
+    - id: mypy
+- repo: https://github.com/pre-commit/mirrors-isort
+  sha: v4.3.20
+  hooks:
+    - id: isort
```

Comment:
you'll want https://github.com/asottile/seed-isort-config otherwise `isort` won't properly classify imports

Reply:
i even looked at it and thought "well, i'm only using 1st party stuff"

Reply:
for now ;)